src/main/java/com/commercetools/project/sync/model/request/ResourceIdsGraphQlRequest.java

```diff
@@ -0,0 +1,78 @@
+package com.commercetools.project.sync.model.request;
+
+import com.commercetools.sync.commons.helpers.GraphQlBaseRequestImpl;
+import com.commercetools.sync.commons.models.GraphQlQueryResources;
+import com.commercetools.sync.commons.models.ResourceKeyIdGraphQlResult;
+import io.sphere.sdk.http.HttpResponse;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.Set;
+
+import static java.lang.String.format;
+import static java.util.Objects.requireNonNull;
+import static java.util.stream.Collectors.joining;
+import static org.apache.commons.lang3.StringUtils.isBlank;
+
+public class ResourceIdsGraphQlRequest extends GraphQlBaseRequestImpl<ResourceKeyIdGraphQlResult> {
```

Comment:
Was trying to understand the difference with `ResourceKeyIdGraphQlRequest` in sync library but the only difference is it's looking for keys, this one for ids 😱  😄

Reply:
Yes, I don't see any difference at all except 'id' for 'key'. Can you make it generic in Sync library? So that we can use the same class here as well?

Reply:
NO 🤔 they might seem the same but totally different things.

Reply:
Okay